packages/neuron-ui/.eslintrc.js

```diff
@@ -96,5 +97,6 @@ module.exports = {
         allowExpressions: true,
       },
     ],
+    "no-multi-spaces": "error",
```

Comment:
This is unnecessary as code formatting is handled by Prettier, so ESLint should focus on handling non-style-related issues or complex style problems that Prettier cannot handle.

Reply:
> This is unnecessary as code formatting is handled by Prettier, so ESLint should focus on handling non-style-related issues or complex style problems that Prettier cannot handle.

I'll move it to the config of `prettier`. Thx.